dart/example_web/web/main.dart

```diff
@@ -79,10 +82,7 @@ void captureException() async {
   } catch (error, stackTrace) {
     print('\nReporting the following stack trace: ');
     print(stackTrace);
-    final sentryId = await Sentry.captureException(
-      error,
-      stackTrace: stackTrace,
-    );
+    final sentryId = await Sentry.captureException(error);
```

Comment:
is the `stackTrace: stackTrace` removed because `error` is an `Error`?

Reply:
It was, but now that I put it back, the captureException fails in release mode.

Reply:
mm, did you figure this out?

Reply:
so after investigations : 
in browser release mode, capturing the error without adding the stacktrace works, but fails with the optional stacktrace if it is passed as stacktrace, but works if it is passes as String :)

 - this works in release mode
```dart
final sentryId = await Sentry.captureException(
      error,
      stackTrace: stackTrace.toString(),
    );
```

- this fails in release mode
```dart
final sentryId = await Sentry.captureException(
      error,
      stackTrace: stackTrace,
    );
```

Reply:
do you have an idea why this happens?

we could add a condition like:

`stackTrace: (stackTrace != null) ? ((isWeb) ? stackTrace.toString() : stackTrace.toString()) : null`

adding a comment why of course.

Reply:
to be more precise without `toString()`it works in chrome, not working in Safari, but I have no idea why.

Reply:
damn thats problematic, let's fix for Chrome and add a comment for Safari

Reply:
as it is it works on Chrome, where would be the better place for a comment ? in the readme ? or StacktraceFactory ?

Reply:
both I'd say, but please create a task for this too, (to research and fix it)

Reply:
✅

Reply:
task was not created, so I did myself
https://github.com/getsentry/sentry-dart/projects/1#card-50215638